test/xml_namespace_test.rb

```diff
@@ -11,27 +11,27 @@
 #     <hr:born>1922-11-26</hr:born>
 #     <hr:dead>2000-02-12</hr:dead>
 #    </hr:author>
-#    <lib:character id="PP">
+#    <hr:character id="PP">
```

Comment:
Why change from `lib` to `hr` ?

Reply:
xmlns:lib="http://eric.van-der-vlist.com/ns/library"
xmlns:hr="http://eric.van-der-vlist.com/ns/person">

Lib is for library and HR for the person

Reply:
Yeah, but `character` is the property of `lib` and not `hr`

Reply:
It possible in the code, but not valid xml, that is why it was failing in jruby since the library in java don't support it.

I can skip this test in jruby